cli/command/service/create.go

```diff
@@ -74,6 +74,8 @@ func runCreate(dockerCli *command.DockerCli, opts *serviceOptions) error {
 
 	ctx := context.Background()
 
+	resolveServiceImageDigest(dockerCli, &service)
```

Comment:
That's true - and it's not a great experience on non-swarm docker atm, where you can't run images that are are built but not pushed.  I lean towards requiring content trust to be turned off, just because it seems like it'd be easier to go from tighter security down, and also because it'd be consistent with docker run behavior, and we probably need to work out a better model for the local image case anyway.

Out of curiosity, `docker build` happens on a single worker node at a time, right?  Does scheduling take into account which nodes have the image locally, or replicate the local images if the service needs to scale?